### DISCUSSION TITLE
没有面板只能自己用 ELK 搭了

### DISCUSSION THREAD

**Discussion by LisonFan** [pymumu/smartdns#1258] [General]
![](https://ghproxy.com/https://raw.githubusercontent.com/xJDt9eAR6K/DgBngiez/mosZckVF/5b85f5e046dee28f490cfe87e0a6c9e4/1675093417333.png)

**Comment by devnakx** [comment]
求教程，我用docker部署的

**Reply by lalasou** [reply]
求指导
